### PR TITLE
styling requests from VW

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR changes some styling of the invoices

**How to prepare for test**:
- [x] install on stage

**How to test**:
- [x] generate a invoice and see that font size, cell hight... has changed. 
- [x] Generate a KI invoice and see that there are two new columns with information about KTH price and total price

**Expected test outcome**:
- [x] A invoice that VW likes

**Review:**
- [x] code approved by @patrikgrenfeldt 
- [x] tests executed by @mayabrandi
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This is minor **version bump**

### CODE REVIEW

**Comment by ingkebil** [comment]
Don't really know what to do with this. Can you let me know when you can show what this does @mayabrandi ?

**Comment by vwirta** [comment]
Let me know if i should review an output document

**Comment by mayabrandi** [comment]
Absolutely. I can show you on Monday!

> 5 dec. 2019 kl. 15:26 skrev Valtteri Wirta <notifications@github.com>:
> 
> Let me know if i should review an output document
>

File: cg/apps/invoice/render.py [hunk 1]
```diff
@@ -65,8 +65,13 @@ def render_xlsx(data: dict) -> Workbook:
                                  bottom=Side(border_style='thin', color='000000'))
             cell.fill = PatternFill('solid', fgColor='E5E8E8')
 
-
     worksheet = workbook['Faktura']
+    worksheet.font = Font(size=14)
+    for row in worksheet.row_dimensions:
+        worksheet.row_dimensions[row].height = 30
```
Comment (patrikgrenfeldt) [inline cg/apps/invoice/render.py]: Could you explain to me how this works?

**Comment by patrikgrenfeldt** [comment]
There is a check failing, can you see if it is fixable? 
image

**Comment by mayabrandi** [comment]
Regarding the testing:
The code has been tested locally, and WV has approved on the new invoice output. Example attached. Problem is that it is not possible to test this on stage due to [this](https://github.com/Clinical-Genomics/cg/issues/537) known issue. 


[Invoice_334_KI.xlsx](https://github.com/Clinical-Genomics/cg/files/4147130/Invoice_334_KI.xlsx)

**Comment by mayabrandi** [comment]
Tests executed on stage. Invoices were generated properly with the new price columns for KI-invoice, and the styling changes. Fontsize etc.. Attracting the results.

[Invoice_285_KTH (10).xlsx](https://github.com/Clinical-Genomics/cg/files/4184708/Invoice_285_KTH.10.xlsx)


[Invoice_285_KI (6).xlsx](https://github.com/Clinical-Genomics/cg/files/4184707/Invoice_285_KI.6.xlsx)

**Comment by mayabrandi** [comment]
I have tested on stage and the invoices were properly generasted with increazed font size and extra price columns for the KI invoice. 
Skärmavbild 2020-02-11 kl  09 46 42

**Comment by patrikgrenfeldt** [comment]
deployed

**Comment by vwirta** [comment]
Thank you guys! 
Will test soon!

**Pull request merged**